app.js

```diff
@@ -314,7 +319,10 @@ app.on('ready', () => {
     }
     // Show changelog if last version wasn't seen
     const lastSeen = store.get('changelog-seen');
-    if (lastSeen !== appVersion) {
+    const lastSeenCount = store.get('changelog-seen-count');
+    const limitChangeLog = store.get('userPrefs.app.analytics.limit-changelog');
+
+    if (lastSeen !== appVersion || (lastSeenCount < maxChangeLogSeenCount && !limitChangeLog)) {
       openSecondaryWindow('changelogWindow');
```

Comment:
Can we please add analytics here with timestamp, time difference between when the window opens and when it's closed. That way we can make a guess to see if the changelog was read or not. Like if its too long, that means user just kept it open, and if its too little that means user just immediately closed it. 

As it's already merged, maybe we can do it in 5.1?